pachyderm/values.yaml

```diff
@@ -1,27 +1,26 @@
+imagePullSecret: "" # FIXME: should this be per-image?
+
 dash:
   image:
     repository: pachyderm/dash
     tag: "0.5.57"
     pullPolicy: IfNotPresent
   # resources: {} TODO
+  enabled: true
 
 pachd:
   image:
     repository: pachyderm/pachd
     tag: "1.12.4"
     pullPolicy: IfNotPresent
-  resources: {}
-    #limits:
-    #  cpu: "1"
-    #  memory: 2G
-    #requests:
-    #  cpu: "1"
-    #  memory: 2G  
+  cpuRequest: 1
+  memoryRequest: 2G
+  guaranteedResources: true # if false, no resource limits set
   logLevel: info
   metrics:
     enabled: true
     endpoint: ""
-  noExposeDockerSocket: false
+  exposeDockerSocket: true
```

Comment:
While I agree this is clearer, I think we should stick with matching the pachd env var for now (We should PR a change to pachd to change it potentially though

Reply:
I really think negative-named flags are confusing, but … I can live with it for the sake of consistency with `pachd`.